README.Rmd

```diff
@@ -0,0 +1,121 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, echo = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-"
+)
+
+# just to keep the code clean below
+suppressWarnings(suppressPackageStartupMessages(library(dplyr)))
+```
+
+# Scales <img src="man/figures/logo.png" align="right" />
+
+[![Build Status](https://travis-ci.org/r-lib/scales.svg?branch=master)](https://travis-ci.org/r-lib/scales)
+[![Coverage Status](https://img.shields.io/codecov/c/github/r-lib/scales/master.svg)](https://codecov.io/github/r-lib/scales?branch=master)
+[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/scales)](https://cran.r-project.org/package=scales)
+
+One of the most difficult parts of any graphics package is scaling, converting from data values to perceptual properties. The inverse of scaling, making guides (legends and axes) that can be used to read the graph, is often even harder! The idea of the `scales` package is to implement scales in a way that is graphics system agnostic, so that everyone can benefit by pooling knowledge and resources about this tricky topic.
+
+# Installation
+
+```{r, eval = FALSE}
+# Scales is installed when you install ggplot2 or the tidyverse.
+# But you can install just scales from CRAN:
+install.packages("scales")
+
+# Or the development version from Github:
+# install.packages("devtools")
+devtools::install_github("r-lib/scales")
+```
+
+# Components
+
+The `scales` package is made up of the following interdependent components
```

Comment:
House style is to _not_ put package names in `` ` ```

Reply:
today I learned... how I missed that thus far not sure 😱